config/locales/en.yml

```diff
@@ -152,10 +152,12 @@ en:
             fonts: "Fonts"
             themes: "Themes"
       sidebar:
+        account: Account
```

Comment:
Should we go ahead and run the translations?

Reply:
That sounds good! I'm not sure how to do that but I'll ask around.

Reply:
there's a playbook article, i'll find and link it

Reply:
It looks like the value of GOOGLE_TRANSLATE_API_KEY is in 1password. Hopefully it's up to date.

Reply:
Great thank you, i'll give that a shot, i remember something about it being out of money but i'm hoping that's been resolved

Reply:
I make another commit for running the translations